client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -10,11 +10,17 @@ import React, { createElement, useRef, FunctionComponent, ReactNode } from 'reac
  * Internal dependencies
  */
 import { Button, ProductIcon } from '@automattic/components';
+import { useLocalizedMoment } from 'components/localized-moment';
 import { preventWidows } from 'lib/formatting';
 import PlanPrice from 'my-sites/plan-price';
 import JetpackProductCardFeatures, { Props as FeaturesProps } from './features';
 import useFlexboxWrapDetection from './lib/use-flexbox-wrap-detection';
 
+/**
+ * Type dependencies
+ */
+import type { Moment } from 'moment';
```

Comment:
I'm not familiar with TS internals. Is it legit to import `moment` this way for types only, @ChaosExAnima?

Reply:
Yup! Or if it isn't, I'm certainly in trouble as I've used this pattern quite a bit! 😅

Reply:
🤞 Thanks!